civicrm_resources/civicrm_resources.module

```diff
@@ -213,3 +51,25 @@ function civicrm_resources_load($extention_full_name, $files = []) {
 function civicrm_resources_flush_caches() {
   return ['cache_civicrm_resources'];
 }
+
+/**
+ * Includes CSS/JS resources onto a page
+ *
+ * @param string $extension_path
+ *   Machine path of civicrm extension.
+ * @param array $resources_uris
+ *   Resource URI relative to extension paths to be included
+ */
+function _civicrm_resources_add_resource($extension_path, $resource_uri) {
+  $resource_type = pathinfo($resource_uri)['extension'];
+  $relative_to_drupal_extension_path = str_replace(DRUPAL_ROOT, '', $extension_path);
+  $relative_to_drupal_resource_path = $relative_to_drupal_extension_path . '/' . $resource_uri;
```

Comment:
To me, these two variable names sound a little bit awkward. How about `extension_path_relative_to_drupal` and `resource_path_relative_to_drupal`? Sound a bit more natural to my ears

Reply:
Done.